### DISCUSSION TITLE
Discussion: Tabs Component Update

### DISCUSSION THREAD

**Discussion by cindyebi** [visual-framework/vf-core#1423] [Proposals]
There is an idea to potentially update the current [vf-tabs](https://latest.visual-framework.dev//components/vf-tabs/index.html?q=tabs) to visually look like the tabs on the Training Academy site for a couple of reasons:
1. More accessible: better contrast with black text against a white background.
2. Consistency with the EMBL Brand guideline by using the interactive Blue color as the "Selected" tab variant.

I'll mockup a few design alternatives this week and possibly add Hover and Disabled variants as well. 

Current:
<img width="1082" alt="Screenshot 2021-03-17 at 14 10 41" src="https://user-images.githubusercontent.com/75834903/111481143-95691480-872a-11eb-8fce-4cdb79355deb.png">

Proposed:
<img width="1132" alt="Screenshot 2021-03-17 at 13 59 38" src="https://user-images.githubusercontent.com/75834903/111479979-6605d800-8729-11eb-9507-16a33cff7333.png">

**Comment by cindyebi** [comment]
@sturobson  I improved the visual element of this new proposed tab component based on the current live tab design on the training site:
• Active state text in Blue instead of Black
• Added a hover effect

![Screenshot 2021-04-28 at 10 01 35](https://user-images.githubusercontent.com/75834903/116377078-be3ff780-a808-11eb-9e7c-3049af0ab417.png)

**Comment by cindyebi** [comment]
@khawkins98 I think we could close this one too?

**Reply by khawkins98** [reply]
One of the things I dislike about github conversations is there's no way to "mark as complete". 

We can lock it though. That seems appropriate — doing it now.
